### PR TITLE
2875: localization issues at poi filter when choosing (münchen - Landeshauptstadt)

### PR DESCRIPTION
**Pull request by bahaaTuffaha**
### Short description

Opening hours Title's Alignment not on the right when arabic is chosen as the device language.

### Proposed changes

- flex-direction is needed when changing the text alignment so I added wrapped the SubTitle with Row component that has a flex-direction already.

Note: the category about education is missing the Arabic translation.

### Side effects

None

### Resolved issues

Fixes: #2875 

---

### CODE REVIEW

File: release-notes/unreleased/2875-Localization-Issues-at-poi-filter.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+issue_key: 2875
+show_in_stores: false
+platforms: # relevant platforms, possible values: web, android and ios
+  - ios
+  - android
+en: Fixed issue with TextAlignment for RTL languages at the opening hours SubTitle
```
Comment (LeandraH) [inline release-notes/unreleased/2875-Localization-Issues-at-poi-filter.yml:6]: 🙃 Can you rephrase that for our users instead of our developers?
But also, I think this should be shown in the stores because it is again something users can notice.

**Review by LeandraH — COMMENTED** [review]
Nicely done! Just the release note needs a fix and then we're good to go :)

File: release-notes/unreleased/2875-Localization-Issues-at-poi-filter.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+issue_key: 2875
+show_in_stores: true
+platforms: # relevant platforms, possible values: web, android and ios
+  - ios
+  - android
+en: Fixed an issue at the map filter where opening hours not aligned correctly
```
Comment (LeandraH) [inline release-notes/unreleased/2875-Localization-Issues-at-poi-filter.yml:6]:
```suggestion
en: Fixed an issue at the map filter where opening hours were not aligned correctly
```

**Review by LeandraH — APPROVED** [review]
Awesome, thank you! Tested in real iOS

**Review by steffenkleinle — APPROVED** [review]
Tested on Android :tada:

**Comment by ztefanie** [comment]
No need to adjust anything, but please never use non standard chars in branch names, could lead to nasty errors:
for ü just use "ue" (for ä use ae, for ö use oe)
